instructor/templates/instructor/wb_band_size.html

```diff
@@ -1,6 +1,9 @@
 {% extends "instructor/assignment_sidebar.html" %}
 
 {% block right_side %}
+    <script>
+    var num_error = {{ num_error }}
```

Comment:
Can you add a semicolon and also change the template variable to `{{ num_error|safe }}`? It's not strictly necessary here since `num_error` is always a number but it's good practice when JSON encoding input to mark it safe so it's not escaped in the template.